packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -23,6 +23,9 @@ const testContext = vm.createContext({
 });
 testContext.global = testContext;
 
+const jest = {};
+jest.expect = global.expect;
```

Comment:
@thymikee Because `expect` is used in the code: https://github.com/babel/babel/pull/5359/files/e6ed1494d94cae8094b890ebe93cbe46f5ec4718#diff-37b0c62f94b613936e9195f14a3320aeR123

I can't remove `chai` because is still used, but maybe we can't remove the `vm` stuff, I'm investigating it